gradle/libs.versions.toml

```diff
@@ -48,7 +48,7 @@ androidXConstraintLayoutCompose = { group = "androidx.constraintlayout", name =
 playServicesMaps = { group = "com.google.android.gms", name = "play-services-maps", version = "19.2.0" }
 playServicesLocation = { group = "com.google.android.gms", name = "play-services-location", version = "21.3.0" }
 playServicesOssLicenses = { group = "com.google.android.gms", name = "play-services-oss-licenses", version = "17.1.0" }
-mapboxAndroidSdk = { group = "com.mapbox.maps", name = "android", version = "10.16.4" } # Check if https://github.com/mapbox/mapbox-maps-android/issues/2389#issuecomment-2432018812 no longer takes place before upgrading
+mapboxAndroidSdk = { group = "com.mapbox.maps", name = "android-ndk27", version = "10.19.0" }
```

Comment:
v10.19.0 seems to be free of that bug (I think it exists in v11+). We should keep the comment so that next time we try to upgrade, we remember to check it.

Reply:
Does that not imply the bug is fixed though? I was thinking we should reply on the issue so they can close it if it's not in that version.

Reply:
Ah, sorry I'd misunderstood. I can add the comment back in.